modules/videopress/utility-functions.php

```diff
@@ -73,13 +73,16 @@ function videopress_get_video_details( $guid ) {
  *
  * Modified from https://wpscholar.com/blog/get-attachment-id-from-wp-image-url/
  *
+ * @deprecated since 8.4.0
```

Comment:
Not required, but nice if we added a `@see` keyword pointing to a close replacement (https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-2-deprecated-functions)

Reply:
Good idea.

Though I couldn't find a possible replacement.

This looks to do the opposite:
https://github.com/Automattic/jetpack/blob/91acb92673f2c44b12cbdac18fb26aa966776369/modules/videopress/utility-functions.php#L663-L671

Reply:
Now we have one!

`video_get_post_id_by_guid`

Reply:
OK, thanks. Applied in https://github.com/Automattic/jetpack/pull/14803/commits/5c05e33d54e8a7987024dc3c7891f54ce6c5a572